AdvancedGuide.md

```diff
@@ -232,11 +232,55 @@ From here on out, if the email is blocked, your user will not be aware. But ther
 
 You may recover your blocked email within 1 month if you email SPEAR (spear@tech.gov.sg) and attach the mail hygiene notification.
 
+## Webhooks
+
+### What are webhooks, and why are they useful?
+
+Webhooks are a standard means for applications like FormSG to send real-time data to other applications and servers. As of May 2020, webhooks are available on Storage mode and can be used by agencies to securely integrate form data collection into the rest of their existing IT workflows. 
+
+Webhooks are useful for agencies who wish to have FormSG response data sent directly to existing IT systems. This removes the need for a human user to periodically export form response data from FormSG, then upload it to an internally-used application.
+
+While webhooks have been tried and tested on high-volume applications, this feature is still in **beta**, meaning users should expect bugs as we work on smoothing out the rough edges. **If you are unable to accept this risk, please do not use this feature until its mainstream release.**
+
+### Who should use webhooks?
+
+Webhooks are worth considering if your agency
+- expects to receive a high volume of responses
+- needs to integrate internal systems/workflows with form data collection (e.g. sorting responses, and channeling them to specific officers for case management)
+- has IT expertise capable of using our software development kit to prepare an endpoint for receiving response data
+
+Webhooks are not necessary or beneficial in most form use cases, where officers are following up over email or Excel spreadsheet, since FormSG already outputs responses in those formats. 
+
+### Are webhooks secure?
+
+Yes - as webhooks are a Storage mode feature, your data is end-to-end encrypted as with all Storage mode responses. Our [software development kit](https://github.com/opengovsg/formsg-javascript-sdk) (SDK) provides instructions on setting up your receiving system to 
```

Comment:
Can link to explanation of end-to-end encrypted in Storage mode section. Most public officers think e2e is simply encryption in transit